src/AltinnCore/Runtime/Controllers/InstanceController.cs

```diff
@@ -278,6 +278,124 @@ public async Task<IActionResult> StartService(string org, string service)
             return View(startServiceModel);
         }
 
+        /// <summary>
+        /// This is the post operation to instansiate an instance
+        /// </summary>
+        /// <param name="startServiceModel">The start service model.</param>
+        /// <returns>JSON Object with the instance ID</returns>
+        [Authorize]
+        [HttpPost]
+        public async Task<dynamic> InstansiateApp(StartServiceModel startServiceModel)
```

Comment:
InstantiateApp (t, not s)

Reply:
Just me making up my own words. And you can't stop me.

... I will change to `InsantiateApp` though.